backend/game/cards/keep_cards/health_manipulation_cards/even_bigger.py

```diff
@@ -0,0 +1,21 @@
+from game.cards.card import Card
+from game.cards.keep_card import KeepCard
+
+
+class EvenBigger(KeepCard):
+    def __init__(self):
+        super(KeepCard, self).__init__("Even Bigger", 4,
+                                         "Your maximum [Heart] is increased by 2. Gain 2[Heart] when you get this card",
+                                         "")
+
+    def immediate_effect(self, player_that_bought_the_card, other_players):
+        player_that_bought_the_card.add_card(self)
```

Comment:
I believe this is something that the professor would rather be handled at the game level rather than having the card manage the player's inventory.

Reply:
removed add/remove card from player logic... agree was bad coupling